pkg/apis/broker/v1beta1/broker_lifecycle_test.go

```diff
@@ -250,76 +264,144 @@ func TestBrokerInitializeConditions(t *testing.T) {
 func TestBrokerConditionStatus(t *testing.T) {
 	tests := []struct {
 		name                string
-		addressStatus       bool
-		brokerCellStatus    bool
-		subscriptionStatus  bool
-		topicStatus         bool
+		addressStatus       string
```

Comment:
It will be better if we can make the UT test code style consistent FYI https://github.com/google/knative-gcp/blob/2f17a5b6dfa268af25e6daa02aed349c65075958/pkg/apis/broker/v1beta1/trigger_lifecycle_test.go#L259
It loo